Epam.TestAutomation/Epam.TestAutomation.Web/PageObjects/Pages/CareersPage.cs

```diff
@@ -0,0 +1,49 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam.TestAutomation.Web.PageObjects.Pages
+{
+    public class CareersPage
+    {
+        private IWebDriver _driver;
+
+        public CareersPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public void ClickCareersLink()
+        {
+            var careersLink = _driver.FindElement(By.XPath("//*[text()='Careers']//ancestor::*[@class='top-navigation__item-text']"));
```

Comment:
Create the separate properties with your elements and then you can reuse them into steps(methods).